lib/render.js

```diff
@@ -48,7 +48,11 @@ if (typeof process.browser === 'undefined') {
 
   // cleanup generated rules in the highlighter registry if they are idle for 2000ms
   // they take a tremendous amount of memory if you process many languages in a server type environment.
-  cleanup(highlighter.registry.grammars)
+
+  // this doesn't work with new highlights version,
```

Comment:
Is `3.1.4` actually newer than `3.2.0-candidate.1`? I think it's not.
Is there any problem keeping the candidate one and keep cleanup code then?

Reply:
Actually 3.1.4 is newer:
<img width="293" alt="Screenshot 2021-01-25 at 21 54 04" src="https://user-images.githubusercontent.com/8243681/106265750-25610680-6230-11eb-81c9-f566cd0591a5.png">

The reason for changing to newer version of highlights is to update `oniguruma` dependency, which is the actual blocker for supporting Node 12.